tutor/controllers/course.py

```diff
@@ -22,3 +23,9 @@ def showCoursePage(id):
         {'id': 3, 'name': 'resource3', 'description': 'blala3', 'link': 'https://google.com'},
     ]
     return render_template('course.html', name=name, resources=resources)
+
```

Comment:
shouldn't resources-related controllers go in a `resources.py` file rather then a `courses.py` file?

Reply:
we were discussing about it. we get into resources from the courses route once we choose a course, so we did it that way. but you're right, we may better do it separately.
@mosesnagar @Michal-Shamir

Reply:
@ifireball I splited the resources related routes and controllers .